Students/Bologa M. Marius - Vasile/Zoowsome/src/javasmmr/zoowsome/models/animals/Aquatic.java

```diff
@@ -0,0 +1,68 @@
+package javasmmr.zoowsome.models.animals;
+
+/**
+ * 
+ * @author Marius Bologa
+ *
+ */
+public abstract class Aquatic extends Animals {
+	/**
+	 * 
+	 */
+	private int avgSwimDepth;
+
+	/**
+	 * 
+	 * @author Marius Bologa
+	 *
+	 */
+	public enum WaterType {
```

Comment:
It would be slightly better to move this enum outside as a separate file. Maybe WaterType.java which just contains this enum (however this is optional)